ui/src/router/components/form/components/ensembler_config/typeOptions.js

```diff
@@ -14,13 +14,26 @@ const typeOptions = [
   {
     value: "standard",
     inputDisplay: "Standard",
+    expenginetype: "standard",
```

Comment:
Added this field so the function `ensemblerTypeOptions` is able to select one of the two `typeOptions` that both have the `value` "standard".